src/Mvc/Mvc.Core/src/Infrastructure/ResourceInvoker.cs

```diff
@@ -1132,7 +1132,7 @@ static async Task Awaited(ResourceInvoker invoker, Task lastTask, State next, Sc
                         {
                             if (_resultExecutingContext == null)
                             {
-                                _resultExecutingContext = new ResultExecutingContextSealed(_actionContext, _filters, _result, _instance);
+                                _resultExecutingContext = new ResultExecutingContextSealed(_actionContext, _filters, _result!, _instance!);
```

Comment:
With the level of `bang` here I'm wondering how much milage we actually get out of this feature. At the minimum, I guess this documents the expectations, so that's a plus and still IMO worth it.

Reply:
We're likely going to have to do this eventually anyway when we turn on Nullable in the csproj.